src/04_printing.py

```diff
@@ -12,6 +12,12 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print("x is %d, y is %f, z is %s." % (x, y, z))
+
 # Use the 'format' string method to print the same thing
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+print("x is {0}, y is {1}, z is {2}".format(x, y, z))
+
+# Finally, print the same thing using an f-string
+
+print(f"x is {x}, y is {y}, z is {z}")
```

Comment:
You got it almost right. Look at the formating within variable z and try to get those quotations marks to print as well